airflow/models/dag.py

```diff
@@ -2755,25 +2755,28 @@ def bulk_write_to_db(
             session.add(orm_dag)
             orm_dags.append(orm_dag)
 
-        # Get the latest dag run for each existing dag as a single query (avoid n+1 query)
-        most_recent_subq = (
-            session.query(DagRun.dag_id, func.max(DagRun.execution_date).label("max_execution_date"))
-            .filter(
-                DagRun.dag_id.in_(existing_dags),
-                or_(DagRun.run_type == DagRunType.BACKFILL_JOB, DagRun.run_type == DagRunType.SCHEDULED),
+        most_recent_runs = {}
+        num_active_runs = {}
+        # Save runtime if dag is not scheduleable
+        if [dag for dag in dags if not isinstance(dag.timetable, NullTimetable)]:
```

Comment:
I think `any([not dag.isinstance(dag.timetable, NullTimeTable) for dag in dags])` syntax is slightly more readable, can you please change it this way?

Reply:
Yep you are right! I changed the code.